File: 백준/Bronze/1546. 평균/평균.js
// dev/stdin
const fs = require("fs");
let input = fs.readFileSync("dev/stdin").toString().split('\n');

const a = Number(input[0]);
const arrNum = input[1].split(' ').map(Number);
const numMax = Math.max(...arrNum);
let sum = 0;



for(let i = 0; i < arrNum.length; i++){
  arrNum[i] = arrNum[i]/numMax*100;
  sum += arrNum[i]
  
}

console.log((sum / a).toFixed(2))

File: 백준/Bronze/1546. 평균/평균.py
import sys

N = int(sys.stdin.readline().strip())

grades = list(map(int,sys.stdin.readline().strip().split(' ')))

maxGrade = max(grades)

for i in range(N):
  grades[i] = grades[i] / maxGrade * 100

print(round(sum(grades) / len(grades), 6))